src/components/logos/Logo.vue

```diff
@@ -1,49 +1,129 @@
 <script setup lang="ts">
-import logo from './logo.svg';
+import fallback from './logo.svg';
+
+interface ExternalLinks {
+  drawer?: string;
+  app?: string;
+  website?: string;
+  about?: string;
+  empty_screen?: string;
```

Comment:
Do you think we can do camelCase as a parameter and use `snakeCase` from `scule` when reading from json?

Reply:
okay, sure thing.